plugin/priv_cc.lua

```diff
@@ -0,0 +1,9 @@
+-- if you need to set a special name for privilege, use "beerchat.name_priv_cc = spec_name" in minetest.conf
+local name_priv2cc = minetest.settings:get("beerchat.name_priv_cc")
```

Comment:
```suggestion
local name_priv2cc = minetest.settings:get("beerchat.priv_cc.name")
```
Just to follow pattern used for other optional extensions: `beerchat.<plugin name>.<setting name>`.

Reply:
Clean.
This is something new for me, thanks for the explanations, I will take note in the future.